packages/story-editor/src/components/library/panes/media/common/innerElement.js

```diff
@@ -244,29 +224,18 @@ function InnerElement({
     handleDrag(resource, event.clientX, event.clientY);
   };
 
+  const thumbnailUrl = type === ContentType.IMAGE ? thumbnailURL : poster;
```

Comment:
Nit: Any other options here instead of using `thumbnailUrl` and `thumbnailURL` as different constants? Can be a bit confusing. Maybe overwrite the `thumbnailUrl` instead?

Reply:
Fixed